cmd/geth/consolecmd_test.go

```diff
@@ -87,7 +87,7 @@ func TestIPCAttachWelcome(t *testing.T) {
 		"--port", "0", "--maxpeers", "0", "--nodiscover", "--nat", "none",
 		"--etherbase", coinbase, "--shh", "--ipcpath", ipc)
 
-	time.Sleep(2 * time.Second) // Simple way to wait for the RPC endpoint to open
+	time.Sleep(30 * time.Second) // Simple way to wait for the RPC endpoint to open
```

Comment:
30 seconds is a long time to wait for tests.

Reply:
Interestingly, in order for the test to pass we needed it to be set at 30 seconds, which seems to vary machine to machine. At 2 seconds, the test fails due to a context time out.